sdk/communication/communication-identity/review/communication-identity.api.md

```diff
@@ -24,6 +24,7 @@ export class CommunicationIdentityClient {
     createUser(options?: OperationOptions): Promise<CommunicationUserIdentifier>;
     createUserAndToken(scopes: TokenScope[], options?: OperationOptions): Promise<CommunicationUserToken>;
     deleteUser(user: CommunicationUserIdentifier, options?: OperationOptions): Promise<void>;
+    exchangeAADtokenForACStoken(token: string, options?: OperationOptions): Promise<CommunicationAccessToken>;
```

Comment:
We discussed it offline that it could have been exchangeAADToken but that might make it confusing.. Also, this is still going to be a private preview we can always get feedback in Arch review

Reply:
The argument can also be renamed to teamsToken